### PR TITLE
EDD example

### LINKED ISSUE #11: EDD Support
Might be a way to make this work w/o nonces. Some documentation in ffraenz/private-composer-installer#22

### PR DESCRIPTION
**Pull request by MWDelaney**
Working Easy Digital Downloads example, fixes #11

### CODE REVIEW

File: .github/workflows/examples/edd.yml [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      # Use EDD API to find download link for latest version, using the EDD item ID (different for every plugin)
+      - name: Get download link from EDD
+        id: EDDResponse
+        uses: fjogeleit/http-request-action@master
+        with:
+          url: '${{ env.VENDOR_URL }}/?edd_action=get_version&item_id=${{ env.PACKAGE_SLUG }}&license=${{secrets.LICENSE_KEY}}&url=${{ env.ACTIVATION_URL }}'
```
Comment (ethanclevenger91) [inline .github/workflows/examples/edd.yml:40]: I believe item_id will accept either the slug or ID, but I would give it the ID - more explicit and more logically matches the param.

Comment (ethanclevenger91) [reply .github/workflows/examples/edd.yml:40]: Alternatively, this example doesn't need both env values and could use only the slug.

File: .github/workflows/examples/edd.yml [hunk 2]
```diff
@@ -7,15 +7,12 @@ on:
   workflow_dispatch:
 
 env:
-  # The plugin slug
+  # The plugin slug (example: searchwp)
```
Comment (ethanclevenger91) [inline .github/workflows/examples/edd.yml:10]: Now that I think about it (sorry) SearchWP is a perfect example of why the slug isn't going to work. When we used the `searchwp` slug, we got version 3. V4 got a new slug (it's `searchwp-4` if you're curious). And that isn't the path we want this installed to, so both are going to be required.

**Review by ethanclevenger91 — CHANGES_REQUESTED** [review]
Use ID explicitly

File: .github/workflows/examples/edd.yml (hunk 2 above)
Comment (MWDelaney) [reply .github/workflows/examples/edd.yml]: You're absolutely right; we already learned that lesson! Fixed.

**Comment by vburlak** [comment]
Hey folks, I've tested it with some other plugin that distributed using EDD. 
Works perfect on my end. 

I hope we could merge this example.
Thanks!

**Pull request merged**